src/usr/lib/rsetup/cli/steam.sh

```diff
@@ -0,0 +1,204 @@
+# shellcheck shell=bash
+get_user_home() {
+    getent passwd "$(logname)" | cut -d: -f6
+}
+
+install_box64() {
+    curl -Ls https://ryanfortner.github.io/box64-debs/box64.list > /etc/apt/sources.list.d/box64.list
+    curl -Ls https://ryanfortner.github.io/box64-debs/KEY.gpg | gpg --dearmor --batch --yes -o /etc/apt/trusted.gpg.d/box64-debs-archive-keyring.gpg
+    apt-get update -y && apt-get install -y box64-rk3588
+}
+
+install_box86() {
+    curl -Ls https://itai-nelken.github.io/weekly-box86-debs/debian/box86.list > /etc/apt/sources.list.d/box86.list
+    curl -Ls https://itai-nelken.github.io/weekly-box86-debs/debian/KEY.gpg | gpg --dearmor --batch --yes -o /etc/apt/trusted.gpg.d/box86-debs-archive-keyring.gpg
+    apt-get update -y && apt-get install -y box86
+}
+
+# shellcheck disable=SC2120
+# files are script files and reference arguments
+install_winex86() {
+    local -r user_home="$(get_user_home)"
+    rm -rf "${user_home}/.wine/" "${user_home}/wine/"
+    # cp wine /usr/local/bin/
+    cat <<\EOF > /usr/local/bin/wine
+#!/bin/bash
+#export GALLIUM_HUD=simple,fps
+setarch linux32 -L box86 $HOME/wine/bin/wine "$@"
+EOF
```

Comment:
```suggestion
    cat <<EOF > /usr/local/bin/wine
#!/bin/bash
#export GALLIUM_HUD=simple,fps
setarch linux32 -L box86 "\$HOME/wine/bin/wine" "\$@"
EOF
```
Should be like this?

Reply:
The \ before `EOF` escapes everything, either way should be ok

Reply:
OK then. I was not aware of this function.